airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3744,7 +3744,24 @@ components:
           description: Alias name to the stream to be used in the destination
           type: string
         selected:
+          description: If this is true, the stream is selected with all of its properties.
           type: boolean
+        fieldSelectionEnabled:
+          description: Whether field selection should be enabled. If this is true, only the properties in `selectedFields` will be included.
+          type: boolean
+        selectedFields:
+          description: Paths to the fields that will be included. This must be set if `fieldSelectedEnabled` is set. An empty list indicates that no properties will be included.
+          type: array
+          items:
+            $ref: "#/components/schemas/SelectedFieldInfo"
+    SelectedFieldInfo:
+      type: object
+      description: Path to a field/column/property in a stream to be selected.
```

Comment:
Maybe an example in this description could help explain what a "path" is, I think it's very unclear without context on nested fields

Reply:
I tried to clarify this a bit; but I think what's called for is a proper snippet of documentation on nested fields in the API, since we use the same convention for cursors and primary keys without any explanation (and it's indeed confusing).